src/main/kotlin/io/github/mojira/arisa/modules/commands/ListUserActivityCommand.kt

```diff
@@ -15,16 +16,16 @@ class ListUserActivityCommand(
     val searchIssues: (String, Int) -> Either<Throwable, List<String>>
 ) {
     operator fun invoke(issue: Issue, userName: String): Int {
-        val escapedUserName = userName.replace("'", "\\'")
+        val issueFunctionInner = escapeIssueFunction(userName) { "by $it" }
 
-        val jql = """issueFunction IN commented("by '$escapedUserName'")
-            | OR issueFunction IN fileAttached("by '$escapedUserName'")"""
-            .trimMargin().replace("[\n\r]", "")
+        val jql = """issueFunction IN commented($issueFunctionInner)
+            | OR issueFunction IN fileAttached($issueFunctionInner)"""
+            .trimMargin().replace(Regex("[\n\r]"), "")
 
         val sanitizedUserName = sanitizeCommentArg(userName)
 
         val tickets = when (val either = searchIssues(jql, ACTIVITY_LIST_CAP)) {
-            is Either.Left -> throw CommandExceptions.CANNOT_QUERY_USER_ACTIVITY.create(sanitizedUserName)
+            is Either.Left -> throw CommandExceptions.CANNOT_QUERY_USER_ACTIVITY(sanitizedUserName, jql).create(Unit)
```

Comment:
Could you please change this to call `initCause` on the created exception with the exception store in the `Either`?
Then we will know why the execution is failing.

Reply:
Ooooh so that's why we didn't see the original cause. Thanks. Yes, will do that.